tools/demobench/src/test/kotlin/net/corda/demobench/pty/JediFXTest.kt

```
@@ -0,0 +1,78 @@
+package net.corda.demobench.pty
```

I think this won't do what you expect - it'll still be a part of every unit test run, just that failures will be ignored? I get that this was useful to write but I'm not sure having it a part of the repo really helps.